singularity-eos/eos/eos_stellar_collapse.hpp

```diff
@@ -158,6 +159,8 @@ class StellarCollapse : public EosBase<StellarCollapse> {
     return 0;
   }
   static inline unsigned long max_scratch_size(unsigned int nelements) { return 0; }
+  // NOTE: The logs here are table logs. SO when fast logs are
+  // enabled, these are fast logs.
```

Comment:
Should not-really-log functions be private methods for internal lookups only? Pretty nasty bug if someone accidentally uses these elsewhere in their own code thinking they are logs.

Reply:
I left these in by accident---meant to remove them. They're already gone from the docs.